modules/preview-react/multi-select/lib/MultiSelectInput.tsx

```diff
@@ -228,34 +186,25 @@ export const MultiSelectSearchInput = createSubcomponent(TextInput)({
           <InputGroup.InnerStart pointerEvents="none" width={system.space.x8}>
             <SystemIcon icon={searchIcon} size={system.space.x4} />
           </InputGroup.InnerStart>
-          <InputGroup.Input data-part="form-input" {...formInputProps} />
+          <InputGroup.Input
+            data-part="form-input"
+            placeholder={null as unknown as string} // do not use a placeholder for this input
+            {...formInputProps}
+          />
           <InputGroup.Input
             data-part="user-input"
             as={Element}
             aria-labelledby={ariaLabelledBy}
             {...elemProps}
           />
-          <InputGroup.InnerEnd>
+          <InputGroup.InnerEnd width={system.space.x4}>
```

Comment:
do we care if this is a style prop or should we use `cs` here?

Reply:
`width` is a CSS variable in inputgroup, so `cs={{width...}}` would now work correctly.